test/support/file_endpoint.ex

```diff
@@ -56,7 +56,9 @@ defmodule Membrane.RTC.Engine.Support.FileEndpoint do
 
   @impl true
   def handle_prepared_to_playing(_ctx, state) do
-    {{:ok, notify: {:publish, {:new_tracks, [state.track]}}}, state}
+    {{:ok,
+      notify: {:publish, {:new_tracks, [state.track]}},
+      notify: {:forward_to_parent, :tracks_added}}, state}
```

Comment:
Why do we need this notification?

Reply:
For HLS tests. They were awaiting the media event previously, but only to make sure that the tracks were added before proceeding